zipline/data/minute_bars.py

```diff
@@ -856,6 +897,17 @@ def last_available_dt(self):
     def first_trading_day(self):
         return self._start_session
 
+    def _ohlcv_ratio_inverse_for_sid(self, sid):
+        if self._ohlc_inverses_per_sid is not None:
+            try:
+                return self._ohlc_inverses_per_sid[sid]
+            except KeyError:
+                pass
```

Comment:
Same as with `ohclv_ratio_for_sid`, should the `_default_ohlc_inverse` be returned here?